### ISSUE TITLE
Plutôt que d'avoir des listes de 25 résultats sur plusieurs pages, peut-on avoir un scroll infini ? 

### ISSUE THREAD

**Issue by GuillaumeAV** [assemblee-virtuelle/semapps#343]
Peut-on avoir un scroll infini ? Plus pratique je trouve ... Ou alors des liste de 50 résultats ?

**Comment by bouviermullerp** [comment]
On peut surement proposer un affichage à 25, 50 et 100 vues au choix. 
Est-ce que ça te conviendrait @GuillaumeAV ?

**Comment by GuillaumeAV** [comment]
Le scroll infini c'est vraiment pas mal en termes d'UX.. J'imagine cependant qu'il y a des enjeux en termes de sobriété.. En alternative, j'aime bien l'approche https://wiki.lowtechlab.org/wiki/Explore?page=2

**Comment by srosset81** [comment]
On peut changer le nombre d'entités

**Comment by GuillaumeAV** [comment]
Question produit, j'ai l'impression, qu'avec la généralisation de l'usage des smartphone notamment, ca penche côté scroll infini, notamment dans le cas de listes d'individus de classes (ce serrait différent pour des articles je pense) mais ce n'est pas hyper fondamental non plus ...

**Comment by bouviermullerp** [comment]
Exact, de toute manière, je propose de fermer cette issue qui me semble trop générique au projet SemApps. Chaque projet qui utilise SemApps aura la liberté de choisir quel type de Scroll il propose, n'est ce pas ?

**Comment by srosset81** [comment]
> Exact, de toute manière, je propose de fermer cette issue qui me semble trop générique au projet SemApps. Chaque projet qui utilise SemApps aura la liberté de choisir quel type de Scroll il propose, n'est ce pas ?

On pourrait proposer un composant de type infinite scrolling, qui serait inclus dans un de nos packages NPM.
A voir ce qu'on garde ici, mais des issues comme https://github.com/assemblee-virtuelle/semapps/issues/29 ou https://github.com/assemblee-virtuelle/semapps/issues/3 me semblent aussi peu "génériques" que celle-ci.

**Comment by bouviermullerp** [comment]
J'aurai du écrire trop *spécifique* pour être dans le projet SemApps. OK pour un composant dans un package.
D'un point de vue stratégique, je suis d'accord qu'il est très important de s'accorder sur ce qu'on garde ici (ce qui est générique), et ce qu'on met dans les projets spécifiques. C'est important pour éviter les malentendus qui peuvent conduire à la frustration.
Les issues #23 et #3 ont été créé au tout début du projet, quand il était à peine défini, c'est donc normal qu'elle sorte du périmètre SemApps si c'est pertinent, aucun problème pour moi.

**Comment by GuillaumeAV** [comment]
Ah oui un composant npm infinite scroll ça semble cool.. En tout cas pour
coographie & archipel, ca reste un sujet..

Le mar. 25 août 2020 à 16:21, Sébastien Rosset <notifications@github.com> a
écrit :

**Comment by bouviermullerp** [comment]
Le croll infini est également une des techniques utilisés dans "l'économie de l'attention", pour garder en ligne plus longtemps le visiteur. CF les explications sur les vidéos dopamine de Arte https://www.youtube.com/watch?v=0PlvSpqKmHY&list=PLCwXWOyIR22vziNWlxEbVsovP-Y8QuIzx
